Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM golang:1.14.4
+ARG GO_VERSION
+FROM golang:${GO_VERSION:-latest}
```

Comment:
A nicer way to solve this would be to use `go get golang.org/dl/go1.15beta1` rename it to `go.rc` or something, then in the jenkinsfiles create a separate test setup with `go.rc` (without race and postgres only)

Reply:
Especially since the docker images are sometimes days behind the actual golang release.